cpu/nrf5x_common/periph/temperature.c

```diff
@@ -24,30 +24,35 @@
 #include "saul_reg.h"
 #include "phydat.h"
 
-void temperature_read(int16_t *temp)
+/**
+ * @brief   Read the temperature in E-02 °C
+ * @return  The measured temperature in E-02 °C (e.g. 1825 would mean 18.25 °C)
+ */
+int32_t temperature_read(void)
```

Comment:
might as well make that `static` if it's not used outside this file

Reply:
Also, `phydat_fit()` seems to be an overkill. With E-02 °C we can still represent temperatures from -327 °C to +327 °C in int16_t, so this never needs to scale anyway.